### DISCUSSION TITLE
[Projects beta] Draft Issues disappear when added on a filtered table.

### DISCUSSION THREAD

**Discussion by danm** [community/community#8983] [Projects and Issues]
I will often use the spreadsheet view with a milestone filter applied. When I create a new draft issue, it will disappear because that newly created asset doesn't have the milestone applied. I then have to go to a different board to change that draft into an issue in order to add the milestone, back to my original filtered view. 

When creating drafts on the table view with a filter applied, if the new draft issue automatically inherits the filters applied to that view, this would avoid them from disappearing.

**Comment by azenMatt** [comment]
We just added assignees to drafts, milestone will also follow in the future

**Reply by danm** [reply]
and labels? 😍

**Reply by HonzaMikula** [reply]
and any other meta data that have single select 🙏🏻
